### PR TITLE
Fix naming clash between attr_accessor and passed arg

### PR DESCRIPTION
**Pull request by lcpriest**

### CODE REVIEW

File: lib/stripe.rb [hunk 1]
```diff
@@ -76,25 +76,36 @@ module Stripe
...
+      api_key_override || @api_key
+    end
   end
 
   def self.api_url(url='', api_base_url=nil)
     (api_base_url || @api_base) + url
   end
 
-  def self.request(method, url, api_key, params={}, headers={}, api_base_url=nil)
+  def self.request(method, url, override_key, params={}, headers={}, api_base_url=nil)
```
Comment (bradleypriest) [inline lib/stripe.rb]: Why the rename?

Comment (lcpriest) [inline lib/stripe.rb]: There is a method called api_key in this file and i dislikehl having them
overlap

Comment (bradleypriest) [inline lib/stripe.rb]: But it is just an api_key



—
Bradley Priest

Comment (lcpriest) [inline lib/stripe.rb]: So I should change it back? I still dislike having the same name as a method

**Pull request merged**